tools/saqc/saqc.xml

```diff
@@ -142,45 +101,65 @@
               <option value="flagConstants">flagConstants: Flag constant data values</option>
             </param>
             <when value="flagByVariance">
-              <param argument="field" type="text" value="" label="Field" help="The name of the variable to process.">
-                <validator type="empty_field"/>
-              </param>
-              <param argument="window" type="text" value="" label="Size of the moving window" help="This is the number of observations used for calculating the statistic. Each window will be a fixed size. If its an offset then this will be the time period of each window. Each window will be sized, based on the number of observations included in the time-period">
+              <param argument="field" type="data_column" label="field" help="Variable to process" data_ref="data" display="checkboxes" multiple="true"/>
+              <param argument="window" type="text" label="window" help="Size of the moving window. This is the number of observations used for calculating the statistic. Each window will be a fixed size. If its an offset then this will be the time period of each window. Each window will be sized, based on the number of observations included in the time-period (Pandas frequency/offset string, e.g., '1D', '2H30M', 'min', 'W-MON')">
```

Comment:
Somehow `label` generation does not work yet as expected. It should be the first sentence.